container/Containerfile

```diff
@@ -60,23 +70,23 @@ ARG libfabric_version=2.0.0
 RUN git clone https://github.com/ofiwg/libfabric.git \
     && cd libfabric \
     && ./autogen.sh \
-    && ./configure --prefix=/usr --with-cuda=/usr/local/cuda --enable-cxi --enable-lnx \
+    && ./configure --prefix=/usr --with-cuda=/usr/local/cuda --enable-cuda-dlopen --enable-gdrcopy-dlopen --enable-cxi --enable-lnx \
     && make -j$(nproc) \
     && make install \
     && ldconfig \
     && cd .. \
     && rm -rf v${libfabric_version}.tar.gz libfabric-${libfabric_version}
 
-RUN wget -q  https://download.open-mpi.org/release/open-mpi/v5.0/openmpi-5.0.6.tar.gz \
-    && tar xf openmpi-5.0.6.tar.gz \
-    && cd openmpi-5.0.6 \
-    && ./configure --with-ofi=/usr --prefix=/usr --with-cxi --with-slingshot \
-       --with-cuda CFLAGS="-lcuda -lnvidia-ml" \
+RUN wget -q  https://download.open-mpi.org/release/open-mpi/v5.0/openmpi-${OMPI_VER}.tar.gz \
+    && tar xf openmpi-${OMPI_VER}.tar.gz \
+    && cd openmpi-${OMPI_VER} \
+    && ./configure --with-ofi=/usr --with-gdrcopy=usr/local --prefix=/usr --with-cxi --with-slingshot \
```

Comment:
Is the relative path in `--with-gdrcopy=usr/local` intended or it's a typo?
The image built successfully so I'm not sure if it's OpenMPI's configure which is being silent about not finding the library or it's working as intended.

Reply:
Yeah it's a typo thanks for catching